submit/templates/submit/file_process.html

```diff
@@ -61,28 +63,30 @@
       <ul>
         <li>Be sure to <span class="has-text-weight-bold">check the compiled preview carefully</span> for any undesired artifacts or errors - especially references, figures, and captions.</li>
         <li><span class="has-text-weight-bold">Check the compiler log</span> for warnings and errors.</li>
+        <li>Double check captions, references and citations.</li>
+        <li>For oversize submissions <em>fully complete your submission first</em>, then request an exception.</li>
       </ul>
-      <!-- <p class="has-text-centered">
-        <a class="button is-link" href="{{ url_for('ui.file_preview', submission_id=submission_id) }}" target="_blank" title="PDF preview opens in a new tab">
-          <span>View PDF</span>
-          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="icon filter-white"><path d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"/></svg>
-        </a>
-      </p> -->
       <p class="has-text-centered"><a class="button" href="{{ url_for('help_texprobs') }}">Help: TeX Rendering Problems</a></p>
     </div>
   </div>
 
   {% elif status == "succeeded" %}
   <div id="status-message" data-status="{{ status }}" class="message is-success">
+    <button class="delete notification-dismiss"></button>
     <div class="message-header">
       <p>
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="icon filter-white" role="presentation"><path d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"/></svg>
         Processing Successful
       </p>
     </div>
     <div class="message-body">
-      <p class="has-text-weight-bold">Be sure to check the compiled preview carefully for any undesired artifacts or errors. Reference numbers and captions are important to check.</p>
+      <p class="has-text-weight-bold">Be sure to check the compiled preview carefully for any undesired artifacts or errors.</p>
       <p>If you need to make corrections, return to the previous step and adjust your files, then reprocess your submission.</p>
+      <p>
+        <strong>Avoid common causes of delay:</strong>
+        Double check captions, references and citations.
```

Comment:
This should be consistent with above suggestion, if you agree with it

Reply:
Done!